File: client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/database/schema/DbSchemaDto.java
package org.evomaster.client.java.controller.api.dto.database.schema;

import java.util.ArrayList;
import java.util.List;

public class DbSchemaDto {

    public DatabaseType databaseType;

    public String name;

    public List<TableDto> tables = new ArrayList<>();

    public List<EnumeratedTypeDto> enumeraredTypes = new ArrayList<>();

    public boolean employSmartDbClean;

    public List<CompositeTypeDto> compositeTypes = new ArrayList<>();
}

File: client-dotnet/src/EvoMaster.Controller.Api/dto/database/schema/DbSchemaDto.cs
namespace EvoMaster.Controller.Api {
    using System.Collections.Generic;

    public class DbSchemaDto {
        public DatabaseType DatabaseType { get; set; }

        public string Name { get; set; }

        public IList<TableDto> Tables { get; set; } = new List<TableDto>();
    }
}